### DISCUSSION TITLE
Creating DOM elements outside of __next div

### DISCUSSION THREAD

**Discussion by Kasea** [vercel/next.js#37117] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

    Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Pro
    Binaries:
      Node: 16.11.0
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 12.1.7-canary.10
      react: 17.0.2
      react-dom: 17.0.2

### Describe the Bug

NextJS SSR's elements outside of __next. Reasoning appears to be related to "invalid DOM".

Note: As of React 18, the bug will have an actual warning, but it's coming from React and not NextJS.
NextJS should look into the bug as a seperate issue, as it might be causing other issues as well.

### Expected Behavior

Everything should be loaded **once** into the __next div.

### To Reproduce

https://github.com/Kasea/nextjs-ssr-bug

**Comment by timneutkens** [comment]
This means you rendered something wrong in your app, e.g. nesting `<a>` in another `<a>` or `<p>` inside of `<a>` and such, rendering invalid HTML causes the devtools to show an invalid order of the DOM. This means that you have a bug in your app.
